pkg/apis/build/v1alpha1/build_types.go

```diff
@@ -54,11 +54,13 @@ type BuildSpec struct {
 	CacheName      string                      `json:"cacheName"`
 	Env            []corev1.EnvVar             `json:"env"`
 	Resources      corev1.ResourceRequirements `json:"resources"`
+	PreviousImage  string                      `json:"previousImage"`
```

Comment:
A convention we are following is to always give an image the key image in an objects spec. 

Could we do something like:
```
lastBuild:
      image: 
```

Reply:
I like the idea. I wasn't a big fan of calling it "PreviousImage" to begin with

Reply:
Also, this will allow us to throw in more metadata here if that might help in follow up builds from an optimization perspective.